helm/aws-ebs-csi-driver-app/values.yaml

```diff
@@ -39,7 +39,6 @@ snapshotController:
   tag: "v4.2.1"
   podLabels: {}
 
-imagePullSecrets: []
```

Comment:
nah this is no good. Seems like this line was part of an array and you removed the leading '-', breaking the syntax

Reply:
ok then :+1: